x-pack/plugins/synthetics/public/legacy_uptime/components/overview/monitor_list/columns/test_now_col.tsx

```diff
@@ -9,20 +9,31 @@ import { EuiButtonIcon, EuiLoadingSpinner, EuiToolTip } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
+import { Ping } from '../../../../../../common/runtime_types';
 import { testNowMonitorAction } from '../../../../state/actions';
 import { testNowRunSelector, TestRunStats } from '../../../../state/reducers/test_now_runs';
 
 export const TestNowColumn = ({
   monitorId,
   configId,
+  selectedMonitor,
 }: {
   monitorId: string;
   configId?: string;
+  selectedMonitor: Ping;
 }) => {
   const dispatch = useDispatch();
 
   const testNowRun = useSelector(testNowRunSelector(configId));
 
+  if (selectedMonitor.monitor.fleet_managed) {
+    return (
+      <EuiToolTip content={PRIVATE_AVAILABLE_LABEL}>
+        <>--</>
+      </EuiToolTip>
+    );
+  }
+
   if (!configId) {
```

Comment:
Could have both confiditons here, then conditionally change the content based on the condition, instead of repeating the same EuiTooltip component.